CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```

Comment:
@rhdolin Would you like to give this a read ? It's stand based on https://www.contributor-covenant.org/ but we should give it a read and see if we required exactly this Code of Conduct.

Reply:
@srgothi92 I'm okay starting with this, and we can adapt if necessary as we go along.